state/my_lists/__tests__/selectors.test.js

```diff
@@ -0,0 +1,56 @@
+import {
+  isMyListsLoading,
+  myListsList,
+  myListsListError,
+  deleteMyListError as deleteMyListErrorSelector
+} from '../selectors';
+
+describe('Movie selectors.', () => {
+  const loading = false;
+  const error = 'Mock error';
+  const deleteMyListError = 'Mock error';
+  const state = {
+    myLists: {
+      loading,
+      error,
+      ids: [1, 2],
+      deleteMyListError
+    },
+    data: {
+      lists: {
+        1: { id: 1, title: 'First mock title' },
+        2: { id: 2, title: 'Second mock title' }
+      }
+    }
+  };
+
+  it('isMyListsLoading() returns loading status', () => {
+    expect(isMyListsLoading(state)).toEqual(loading);
```

Comment:
You can use .toBeFalsy()
https://jestjs.io/docs/en/expect#tobefalsy

Reply:
Fixed